client/src/main/java/lighthouse/subwindows/ShowPledgeWindow.java

```diff
@@ -54,6 +59,13 @@ private void init(Project project, LHProtos.Pledge pledge) {
         // Let the user save their pledge again if it's a serverless project and this pledge is ours.
         LHProtos.Pledge pledgeFor = Main.wallet.getPledgeFor(project);
         saveToFileButton.setVisible(project.getPaymentURL() == null && pledgeFor != null && LHUtils.hashFromPledge(pledge).equals(LHUtils.hashFromPledge(pledgeFor)));
+        
+        // Load localized strings
+        saveToFileButton.setText(_("Save to file"));
+        closeButton.setText(_("Close"));
+        pledgedByLabel.setText(_("pledged by"));
```

Comment:
I suspect the way I've inserted controls into the middle of the text here also doesn't translate well into all languages. We might need to write a bit of code to re-order the controls as appropriate here instead. I'll leave this for now.